src/config/index.ts

```diff
@@ -1,228 +1,166 @@
-import memoize from 'lodash/memoize'
+import { ExplorerButton } from '@gnosis.pm/safe-react-components'
```

Comment:
I am only using it to extract the types for consistency: `Parameters<typeof ExplorerButton>[0]['explorerUrl']`. It's a strange props either way because it accepts a function that returns an object.

I think it may be best to change it to just accept and object, or even a string. The `alt` was never used. What do you think?